### PR TITLE
Adds provider text to siteBanner

### PR DESCRIPTION
**Pull request by ryanml**
Fixes: https://github.com/brave/brave-browser/issues/2238

Please use this native ledger PR to test: https://github.com/brave-intl/bat-native-ledger/pull/186

brave-ui update: https://github.com/brave/brave-ui/pull/294

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Windows
  - [ ] macOS
  - [ ] Linux
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on
  - [ ] Windows
  - [x] macOS
  - [ ] Linux
- [x] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone

## Test Plan:
Defined in issue, see: https://github.com/brave/brave-browser/issues/2238

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

### CODE REVIEW

File: components/brave_rewards/resources/donate/components/siteBanner.tsx [hunk 1]
```diff
@@ -119,6 +120,7 @@ class Banner extends React.Component<Props, State> {
       publisherKey = publisher.publisherKey
       description = publisher.description
       name = publisher.name
+      provider = publisher.provider
```
[the remark this answers is not in the document]
Comment (ryanml) [reply components/brave_rewards/resources/donate/components/siteBanner.tsx]: @jasonrsadler fixed, I forgot to add it to siteBanner once ui was updated :)

**Comment by jasonrsadler** [comment]
Text missing expected?
screen shot 2018-12-02 at 11 44 42 pm

**Comment by ryanml** [comment]
@jasonrsadler forgot to add the locale to "donate", fixed

**Comment by jasonrsadler** [comment]
master (0.60.x): d3d7e3a

**Pull request merged**